lib/esyBuildRuntime.sh

```diff
@@ -70,20 +70,22 @@ esy-copy-permissions () {
 }
 
 esy-replace-string () {
-  INPUT_FILE="$1"
-  OUTPUT_FILE="$2"
-  SRC_STRING="$3"
-  DEST_STRING="$4"
-  # TODO: This uses bbe command which is not available by default on linux,
-  # darwin at least.
-  bbe -e "s|$SRC_STRING|$DEST_STRING|" -o "$OUTPUT_FILE" "$INPUT_FILE"
+  FILE="$1"
+  SRC_STRING="$2"
+  DEST_STRING="$3"
+  # TODO: get rid of python here
+  python -c "
```

Comment:
This introduces dependency on `python` which is not so bad in terms of availability (most systems has it out of the box) but bad in terms of speed.